scilpy/tractanalysis/features.py

```diff
@@ -35,15 +35,14 @@ def remove_loops_and_sharp_turns(streamlines,
     -------
     A tuple containing
         list of ndarray: the clean streamlines
-        list of ndarray: the list of removed streamlines, if any
+        list of ndarray: the ids of clean streamlines
     """
```

Comment:
Considering that the function signature hasn't changed, but the output yes. People might not see their use is deprecated. Should we print some warning?

Reply:
I don't know; I've changed every reference to this function in scilpy to that the return matches. I don't know if many people use scilpy functions outside the lab and would not know about this change. I feel like a deprecation warning would be overkill